svc-bip-api/build.gradle

```diff
@@ -6,8 +6,10 @@ plugins {
 
 ext {
   jacocoExclude = ['**/*model*']
+  jacoco_minimum_coverage = 0.5
```

Comment:
Alternatively, we can just ignore the entire config package: `jacocoExclude = ['**/*model*', '**/config']`. No need to test the spring config files.

Reply:
Sounds good! I will exclude the config folder and remove the minimum coverage %